src/main/resources/init_db.sql

```diff
@@ -2,19 +2,45 @@ CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
 USE `taxi`;
 
 DROP TABLE IF EXISTS `manufacturers`;
-CREATE TABLE `manufacturers` (
-                                        `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
-                                        `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+CREATE TABLE `manufacturers`
+(
+                            `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                            `name` VARCHAR(225) NOT NULL,
+                            `country` VARCHAR(225) NOT NULL,
+                            `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                            PRIMARY KEY (`id`)
+);
 
 DROP TABLE IF EXISTS `drivers`;
-CREATE TABLE `drivers` (
-                                  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                  `name` VARCHAR(225) NOT NULL,
-                                  `license_number` VARCHAR(225) NOT NULL,
-                                  `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                  PRIMARY KEY (`id`),
-                                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
-                                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+CREATE TABLE `drivers`
+(
+                            `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                            `name` VARCHAR(225) NOT NULL,
+                            `license_number` VARCHAR(225) NOT NULL,
+                            `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                            PRIMARY KEY (`id`),
+                            UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
+                            UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE
+);
+DROP TABLE IF EXISTS `cars`;
+CREATE TABLE `cars`
+(
+                            `id` BIGINT NOT NULL AUTO_INCREMENT,
+                            `model` varchar(255) NOT NULL,
+                            `manufacturer_id` BIGINT NOT NULL,
+                            `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                            PRIMARY KEY (`id`),
+                            KEY `manufacturer_id` (`manufacturer_id`),
+                            CONSTRAINT `manufacturer_id_fk` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
+);
+DROP TABLE IF EXISTS `cars_drivers`;
```

Comment:
all of these scripts won't work if there is any data in ur tables because u cant drop the table until any other is referencing on its fields